### ISSUE TITLE
[[Bug]] Infobox Formatting

### ISSUE THREAD

**Issue by RinVindor** [rlach/wiki-import#10]
I'm not 100% sure its a bug but I figured I'd check because I could have sworn it imported the tables as they appear in the wiki but here is a screenshot of what I'm getting when I import.

Edit: Also worth noting I have the proper infobox listing and its capitalized correctly.

![image](https://user-images.githubusercontent.com/12647469/188699559-5b8dcf19-c2b7-4bd4-9e87-bfe4c6f10b9d.png)

**Comment by rlach** [comment]
Looking into the code, it appears this table looks like this:
```
| caption            = Map showing Triboar and environs.
| name               = Triboar
| alt spelling       = 
| aliases            = Town Where Only Gwaeron Sleeps<ref name="VoloN-p85" /><ref name="NGSF.CC-p36" />The Gateway to the North<ref name="PotA-p18" />
```
So why the infobox doesn't it show "Geography" section, while it does on wiki?

Most wikis define their templates, in which the data is then put in. This is the template of Location Infobox: https://forgottenrealms.fandom.com/wiki/Template:Location

As you can see, it's huge, has a lot of functionality, and showing the Infobox exactly as on wiki would mean we would have to parse the template first(and download it first). Consider there are sometimes multiple templates on a page, and I'll bet even templates within templates, and it becomes very bothersome.

The template is also responsible for knowing that `aliases` should be shown as `Aliases`, or maybe on another wiki it would be `List of all aliases`. We can't know without the template, and some properties I've seen are... very user unfriendly.

This, and the underlying library [wtf_wikipedia](https://github.com/spencermountain/wtf_wikipedia) doesn't do that. Just as the library readme says - there's hundreds of thousands templates on the English wikipedia alone. This would have been ungodly amount of work for a little prettier tables.

I've also noticed horizontal bars from sections go over the table and images, I reported it to Foundry https://github.com/foundryvtt/foundryvtt/issues/8116 

TLDR: won't do unless wtf does it(which they most probably won't).

**Comment by RinVindor** [comment]
Gotcha! Thanks I was actually going to report the weird h3 horizontal bar format issue. 

Yeah I was pretty sure this one was out of your hands but I wanted to make sure.
